core-services/openshift-monitoring/cfg_prometheus-prow-rules_prometheusrule.yaml

```diff
@@ -48,7 +48,7 @@ items:
 
       - alert: ETCD_OOMKilled_Soon
         expr: |
-          sum(container_memory_rss{namespace=~"kube-system",pod_name=~"master-etcd.*",container_name="etcd"}) > 18000000000
+          container_memory_rss{namespace=~"kube-system",pod_name=~"master-etcd.*",container_name="etcd"} > (2 ^ 34)
```

Comment:
Sorry, I can't do this in my head. Can you please write the number out?

Reply:
Expanded the three `2 ^ 10` factors.